cmd/skupper/skupper.go

```diff
@@ -22,6 +22,28 @@ import (
 
 var version = "undefined"
 
+const (
+	bash_completion_func = `# $1 is the name of resource (required)
+# $2 is template string for kubectl get (optional)
+__kubectl_parse_get()
```

Comment:
What should the assumption be in an openshift environment (e.g. oc might be present but not kubectl)?

Would adding list to pkg/kube/namespace be an alternative?

Reply:
basically to auto-complete, bash executes "some executable", it may be anything, in this case we are executing "kubectl -n ..." we could execute "skupper -something" or build a particular new skupper executable like "skupper-completer" orrr whatever option we want.
Will take a look now if from bash I can "try" with "oc" and if fails, then try with "kubectl", which I think it is totally possible.